smoketest.py

```diff
@@ -26,11 +26,11 @@
 
 
 def validate_args():
-    # Ensure that URL is valid and PAT is well formed
-    # TO DO
+    if ((target[0:8] == "https://") and (target[-4] == ".") and (target[-3:].isalnum())):
```

Comment:
Good intentions, but an easy trap to fall into 😢 

`http://foo.tech` is a valid URL as far as connectivity is concerned, but would this check on both the `target[0:8]` and `target[-4] == "."` checks.

I took a shot at this as well in https://github.com/maclarel/ghes-smoketest/blob/refactor_1/smoketest.py#L30-L32... Not sure if it's better/worse, but _may_ help cover some of the fun of variable length TLDs and non-TLS protocol.

Thoughts?

Reply:
totally forgot about 4+ character stuff, pulled your change into mine